src/command/list/human.rs

```diff
@@ -41,14 +50,55 @@ pub(super) fn format(toolchain: &Toolchain) -> Option<String> {
 /// Accepts the components *from* the toolchain rather than the item itself so
 /// that
 fn display_active(
-    node: &Option<Node>,
-    package_manager: &Option<PackageManager>,
+    runtime: &Option<Box<Node>>,
+    package_manager: &Option<Box<PackageManager>>,
     packages: &[Package],
 ) -> String {
-    match (node, package_manager, packages) {
-        (None, _, _) => NO_RUNTIME.to_string(),
-        (Some(node), Some(package_manager), packages) => unimplemented!(),
-        _ => unimplemented!(),
+    match (runtime, package_manager) {
+        (None, _) => NO_RUNTIME.to_string(),
+        (Some(runtime), Some(package_manager)) => {
+            let runtime_version: String =
+                WRAPPER.fill(&format!("Node: {}", format_runtime(runtime)));
+            let package_manager_version: String = WRAPPER.fill(&format!(
+                "Yarn: {}",
+                format_package_manager(package_manager)
+            ));
+            let package_versions = if packages.is_empty() {
+                WRAPPER.fill(&format!("Tool binaries available: NONE"))
+            } else {
+                WRAPPER.fill(&format!(
+                    "Tool binaries available:\n{}",
+                    format_tool_list(packages)
+                ))
+            };
+
+            format!(
+                "⚡️ Currently active tools:\n\n{}\n{}\n{}\n\n{}",
+                runtime_version,
+                package_manager_version,
+                package_versions,
+                "See options for more detailed reports by running `volta list --help`."
+            )
+        }
+        (Some(runtime), None) => {
+            let runtime_version: String =
+                WRAPPER.fill(&format!("Node: {}", format_runtime(runtime)));
+            let package_versions = if packages.is_empty() {
+                WRAPPER.fill(&format!("Tool binaries available: NONE"))
+            } else {
+                WRAPPER.fill(&format!(
+                    "Tool binaries available:\n{}",
+                    display_packages(packages)
```

Comment:
This block uses `display_packages` to show the package list, but the block above uses `format_tool_list` in the exact same place. Is there a need for these two to be different?